Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -5308,6 +5309,39 @@ def com_google_fonts_check_stylisticset_description(ttFont):
         yield PASS, "OK"
 
 
+@check(
+    id="com.google.fonts/check/os2/fsselectionbit7",
+    rationale="""
+    All fonts should have OS/2.fsSelection bit 7 (USE_TYPO_METRICS) set \
+    so that they use typo vertical metrics instead of Win vertical metrics.
+    """,
```

Comment:
For CJK families, we don't want this bit enabled. Could you add the following condition.

```
conditions = ['not is_cjk_font']
```

Reply:
Np.  Is there a CJK family that I can add to the tests?

Reply:
You could use https://github.com/googlefonts/fontbakery/tree/main/data/test/cjk

Reply:
Added in 0922e2c38a4c84220bb7c01bf2a8a921fba23821 but the test doesn't confirm the skip for the CJK font in the data dir.  Should that font be detected as a CJK font? If so, am I doing something wrong with the condition definition?

Reply:
I'll take a look. Thanks, @chrissimpkins!